### DISCUSSION TITLE
Location for Screenshots (eg: OneDrive users...)

### DISCUSSION THREAD

**Discussion by NachoVega80** [M4elstr0m/TarkovMapTracker#2] [Bugs] [answered]
Hello, I have a problem with the software not accessing my screenshot folder but opening up enclosing documents folders. I think that's the reason why the software does not show aixh a current location? Can you help me find out why I don't see the current location?

**Comment by M4elstr0m** [comment]
Sure lets find out!
Do you know where your screenshot folder is located or not? It should be located in C:/Users/USERNAME/Documents/Escape From Tarkov/Screenshots if it is not, please can you tell me where it is located?

**Reply by NachoVega80** [reply]
Hello. thank you for answering. i'm only back in my home tomorrow evening. i'll send you my folder directory then. until then i wish you a nice Christmas:)

**Reply by M4elstr0m** [reply]
Thanks a lot! Merry Christmas to you as well ;)

**Comment by NachoVega80** [comment]
here is my folder. :) 

C:\Users\MIFCOM\OneDrive\Dokumente\Escape from Tarkov\Screenshots

**Reply by NachoVega80** [reply]
really thank you for your service :)

**Reply by NachoVega80** [reply]
perhaps it is easier in future to let the user choose the screenshot folder by himself :). i use onedrive and it seems that onedrive is changing the default folderpath.

**Reply by M4elstr0m** [reply]
> here is my folder. :)
> 
> C:\Users\MIFCOM\OneDrive\Dokumente\Escape from Tarkov\Screenshots

Effectively the issue is caused by OneDrive but you're right, user should be able to change the screenshot folder path ;) I will work on a new version as soon as I can (I may be able to finish it in less than one week)
I'll contact you here as soon as it is done, thanks a lot for your help 😎

**Comment by NachoVega80** [comment]
ok thank you very much. i will try to find out a way to move the screenshot Folder in any other way ..... perhaps i can find a way.

**Reply by M4elstr0m** [reply]
Don't hesitate to tell me if you manage to change your screenshot folder ;) but I will start working on a new version for you tomorrow 😉

**Reply by NachoVega80** [reply]
hello do you already make any progress wioth the new version? :)

**Reply by M4elstr0m** [reply]
Yup it should be ready later today 😎 sorry for the waiting, it took some time because of the winter events 🎄

**Reply by M4elstr0m** [reply]
Alright, I updated the Tarkov Map Tracker for OneDrive users, you can't manually set your Screenshot folder but it detects it! (Maybe it will be possible in the future, but for now it should be efficient).

Please can you try the TarkovMapTracker and tell me if it works? 😉 You can download it here [Download From Github](https://github.com/M4elstr0m/TarkovMapTracker/archive/refs/heads/main.zip)

Thanks a lot @NachoVega80 !

**Comment by NachoVega80** [comment]
perfect. i will test it now

**Reply by NachoVega80** [reply]
is it possible to use another map picture? Perhaps a pic from the mapgenie map ?!

**Reply by M4elstr0m** [reply]
> perfect. i will test it now

Does it work? :)

**Reply by M4elstr0m** [reply]
> is it possible to use another map picture? Perhaps a pic from the mapgenie map ?!

Technically, you can screenshot mapgenie and replace the map in the assets folder of the TarkovMapTracker, but Mapgenie maps:
- Are not free of use (my maps are from tarkov.dev)
- Might not be super accurate
- May not work at all if the map is rotated/totally different from my version

So you can still try ;)

Can you tell me what do you search from Mapgenie? Is it loot location? PMC spawns?

I'm not planning to switch my maps to Mapgenie (for copyright reasons), but I plan to update my current maps with more data in the future using tarkov.dev

Thanks a lot ;)

**Comment by NachoVega80** [comment]
Hello. thank you very much for the new Version. But it still doesnt work for me. I dont see any red Button who shows my position. I downloaded the new Version. Installed it and opened the exe. After taking a screenshot and cklicking the Locate Button ... nothing happens. Ohter Versisions like The Tarkov Pilot is working for me . so i am sure that i am doing everythign right. or what can i do wrong?

**Reply by M4elstr0m** [reply]
Hello! Thank you for your answer! I think the problem with OneDrive is not solved yet 👀 I will need your help to test a few things in order to understand:
- When you press the "Folder" button from the Tarkov Map Tracker, does it open the good folder? (in OneDrive) if not please tell me which folder it opens (this is the most important information to me)
- If not, do you have a folder "Escape From Tarkov" then "Screenshots" in your Documents folder (the one not in OneDrive)? If it is the case (so if you have 2 Screenshots folder in fact) try to delete the Escape From Tarkov folder that is outside OneDrive, then try the Tarkov Map Tracker again

This issue is a must-fix for me so I want to fix it as soon as I can 😎

**Comment by NachoVega80** [comment]
ok here we go: 

When i push the folder button --> it opens this folder: C:\Users\MIFCOM\OneDrive
My Screenshots are in: C:\Users\MIFCOM\OneDrive\Dokumente\Escape from Tarkov\Screenshots
And i have only this one screeenshot folder :)

**Reply by M4elstr0m** [reply]
Perfect I think I found our issue!
Since it opens your OneDrive folder successfully, that means my new detection system works... However, it seems my software does not handle languages properly 😆 

I will fix that tomorrow, since it won't take much time to fix

Thanks for your help, I will tell you when I upload the fix 😎

**Comment by preynolds2** [comment]
I have the same problem.
Nothing is showing up on the map when I click Locate.

When I click the folder icon..  it shows my path as "C:\Users\NameHere\OneDrive\Documents"

Actual tarkov sceenshots are saved to "C:\Users\NameHere\OneDrive\Documents\Escape from Tarkov\Screenshots"

Althogh I thought I cured my computer from using OneDrive... I guess I didnt.  But.. a totally different problem.  ;)

**Reply by M4elstr0m** [reply]
Oh well I thought this would be a problem of language but if your Documents folder is already "Documents" and it does'nt work then I have a much bigger issue... 
I'm working on it right now thank you for your feedback!

**Comment by M4elstr0m** [comment]
Hey @NachoVega80 and @preynolds2 !

I did an update just now for the Tarkov Map Tracker that finally allows the user to choose their own path for screenshots. (So this update removes the OneDrive auto-detect feature as well)
This is a test version with the minimum required (not included in the ReadMe.md yet, and no UI) let me explain:

In your main folder of the Tarkov Map Tracker's new update, you will find a "forcepath.txt" file. Please paste your Screenshots folder path in it (at the first line) like "C:\Users\NameHere\OneDrive\Documents\Escape from Tarkov\Screenshots" but without the quotes of course, then choose your current map take a screenshot (or set up the software depending on the last screen you made)

Please can you try again the TarkovMapTracker and tell me if it works? 😉 You can download the new version here [Download From Github](https://github.com/M4elstr0m/TarkovMapTracker/archive/refs/heads/main.zip)

I tried to test it using a OneDrive path, it worked, but I prefer to hear it from you 👀

Thanks a lot to both of you!

**Reply by killloard** [reply]
Just downloaded, did what you said for force locate, and worked

**Reply by M4elstr0m** [reply]
Perfect! Thank you for your feedback 😉 don't hesitate to star this repository if you find the Tarkov Map Tracker useful ⭐

**Comment by M4elstr0m** [comment, marked as answer]
The `forcepath` feature is now fully integrated in the upcoming TarkovMapTracker [v2.0.0](https://github.com/M4elstr0m/TarkovMapTracker/releases/tag/v2.0.0)! Please refer to README.md
